README.md

```diff
@@ -0,0 +1,64 @@
+# to-do-list-draggable
+A smart draggable to do list project copy to practice code review and refactoring
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot_moblile.png)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://henrykc24.github.io/to-do-list-draggable-review/dist/)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+Clone the repo with `git clone https://github.com/HENRYKC24/to-do-draggable-review.git`
```

Comment:
Dear Henry, here is a link to clone is not as a repo name, it should be "to-do-list-dragable-review" with ..list..

Reply:
Thank you so much @elmar8287 for your effort in making my project better.
Could you please explain more on the above point:pray:

Reply:
Thank you for clarifying this in the zoom call. I am really grateful:pray: